web/src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -237,9 +240,10 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
 
   handleSubmit = async () => {
     const { onSubmit } = this.props
-    const { selectedFeedbackOption, comment } = this.state
+    const { selectedFeedbackOption, comment, contactMail } = this.state
     try {
-      await this.postFeedbackData(this.getFeedbackData(selectedFeedbackOption, comment))
+      await this.postFeedbackData(
+        this.getFeedbackData(selectedFeedbackOption, `${comment}    contactMail: ${contactMail}`))
```

Comment:
```suggestion
        this.getFeedbackData(selectedFeedbackOption, `${comment}    contactMail: ${contactMail || 'Keine'}`))
```

Reply:
If there is no contactMail then add a meaningful word instead of an empty string.